lib_eio/net.mli

```diff
@@ -171,6 +172,12 @@ val recv : #datagram_socket -> Cstruct.t -> Sockaddr.datagram * int
     returned along with the sender address and port. If the [buf] is too small then excess bytes may be discarded
     depending on the type of the socket the message is received from. *)
 
+(** {2 DNS queries} *)
+
+val addrinfo: #t -> string -> ([`Tcp | `Udp] * Ipaddr.v4v6) list
```

Comment:
```suggestion
val addrinfo: #t -> string -> Sockaddr.t list
```
(or at least a compatible subset, if you want to exclude Unix-domain sockets)

Reply:
Ended up using `Sockaddr.t` as suggested.